src/avram/save_operation.cr

```diff
@@ -260,7 +260,7 @@ abstract class Avram::SaveOperation(T)
         @@permitted_param_keys << "{{ attribute_name.id }}"
       {% else %}
         {% raise <<-ERROR
-          Can't permit '#{attribute_name}' because the column has not been defined on the model.
+          Can't permit '#{attribute_name}' because the column has not been defined on the model #{@type}.
```

Comment:
This isn't quite right since `@type` isn't a model, it's the SaveOperation class. To test the error message, look for any SaveOperation in the specs, and add any random word to the `permit_columns` like `banana` or something. Then try to run the specs, and you'll see the error message.

This will need to say something more like `defined on the model for SaveUser`

Reply:
![avram-SaveUser-error](https://user-images.githubusercontent.com/22129063/131960864-f36c45e3-3a6a-4af2-a101-7d19b5057a9e.png)
@jwoertink Is this a valid error? Or do I need to make any changes?  Could you please guide me a little?
Thanks

Reply:
That's a good error 👍

Reply:
I have made the changes. Please take a look at it and let me know if this needs any changes.
Thanks